tests/test_toml_formatter.py

```diff
@@ -67,12 +67,12 @@ def test_formatter_rule(self):
     def test_formatter_deep(self):
         """Test that the data remains unchanged from formatting."""
         self.compare_test_data(self.test_data[1:])
-
-    def test_format_of_all_rules(self):
-        """Test all rules."""
-        rules = rule_loader.load_rules().values()
-
-        for rule in rules:
-            is_eql_rule = rule.type == 'eql'
-            self.compare_formatted(
-                rule.rule_format(formatted_query=False), callback=nested_normalize, kwargs={'eql_rule': is_eql_rule})
+    #
+    # def test_format_of_all_rules(self):
+    #     """Test all rules."""
+    #     rules = rule_loader.load_rules().values()
+    #
+    #     for rule in rules:
+    #         is_eql_rule = isinstance(rule.contents.data, EQLRuleData)
+    #         self.compare_formatted(
+    #             rule.rule_format(formatted_query=False), callback=nested_normalize, kwargs={'eql_rule': is_eql_rule})
```

Comment:
```python
from detection_rules import rule_loader
from detection_rules.rule import EQLRuleData
```

```suggestion

    def test_format_of_all_rules(self):
        """Test all rules."""
        rules = rule_loader.load_rules().values()

        for rule in rules:
            is_eql_rule = isinstance(rule.contents.data, EQLRuleData)
            self.compare_formatted(
                rule.contents.to_dict(), callback=nested_normalize, kwargs={'eql_rule': is_eql_rule})
```

Reply:
i changed this a different way, so i think we're good now 🤞